src/main/java/com/jcabi/github/Releases.java

```diff
@@ -66,6 +66,25 @@
     Release get(int number);
 
     /**
+     * Get a release by tag name.
+     * @param name Release tag name
+     * @return Release
+     * @throws java.io.IOException If there is any I/O problem
+     * @see <a href="https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name">Get a release by tag name</a>
+     */
+    Release tagged(
+            String name)
```

Comment:
@weisebrazil please take a look at other methods in the project, here everything will fit in one line (maybe the throws will move to the next one if it exceeds 80 column)